### DISCUSSION TITLE
Is there a way to avoid square brackets from being treated as html tags?

### DISCUSSION THREAD

**Discussion by jrod95** [add2cal/add-to-calendar-button#381] [Q&A] [answered]
Hello everyone,

I was wondering if there is a way to avoid square brackets from being read as an html tag? I encountered this issue yesterday using the url tag with a url that had multiple square brackets.

**Comment by jekuer** [comment, marked as answer]
Yes. You can use curly brackets instead. 😃

**Reply by jrod95** [reply]
Thanks @jekuer !
